solutions/go/01-a01/explanation.md

```
@@ -0,0 +1,88 @@
+		var err error
+		fenStr := request["fen"].(string)
+		fen, err = chess.FEN(fenStr)
+		if err != nil {
+			fenStr += " 0 1"
+			fen, err = chess.FEN(fenStr)
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusBadRequest)
+				return
+			}
+		}
+		fmt.Println(fen)
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)